mopidy_youtube/apis/youtube_bs4api.py

```diff
@@ -362,14 +370,13 @@ def list_related_videos(cls, video_id):
             results = soup.find_all("li", class_=["related-list-item"])
             if not results:
                 logger.info("nothing in the soup, trying japi related videos")
-                json_regex = r'window\["ytInitialData"] = ({.*?});'
-                extracted_json = re.search(json_regex, result.text).group(1)
-                result_json = json.loads(extracted_json)["contents"][
-                    "twoColumnWatchNextResults"
-                ]["secondaryResults"]["secondaryResults"][
-                    "results"
-                ]  # noqa: E501
-                items = jAPI.json_to_items(cls, result_json)
+
+                yt_data = cls._find_yt_data(result.text)
+                json = yt_data["contents"]\
```

Comment:
Calling this `json` breaks things; better to call it `result_json` or similar, as previously, I think.